packages/svelte/src/compiler/phases/patterns.js

```diff
@@ -23,3 +23,5 @@ export const regex_heading_tags = /^h[1-6]$/;
 export const regex_illegal_attribute_character = /(^[0-9-.])|[\^$@%&#?!|()[\]{}^*+~;]/;
 export const regex_bidirectional_control_characters =
 	/[\u202a\u202b\u202c\u202d\u202e\u2066\u2067\u2068\u2069]+/g;
+export const regex_js_code = /^\W*javascript:/i;
```

Comment:
```suggestion
export const regex_js_prefix = /^\W*javascript:/i;
```
clarifies the purpose of the regex